service/src/main/java/org/eclipse/keti/acs/service/policy/evaluation/PolicyEvaluationServiceImpl.java

```diff
@@ -126,6 +113,13 @@ public PolicyEvaluationResult evalPolicy(final PolicyEvaluationRequestV1 request
         if (null == result) {
             result = new PolicyEvaluationResult(Effect.NOT_APPLICABLE);
 
+            LinkedHashSet<PolicySet> filteredPolicySets = filterPolicySetsByPriority(subjectIdentifier, uri,
+                    policySetsEvaluationOrder);
+
+            if (filteredPolicySets.isEmpty()) {
```

Comment:
This should never be the case. Since `filterPolicySetsByPriority` fails if a policySet specified does not exist.  Also, its more readable to not have this code, since the for loop on line 136 will be a no-op anyway.